### PR TITLE
Update README.org

### PR DESCRIPTION
**Pull request by dhruvdutt**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -50,7 +50,7 @@ There's two methods.
    overwrite it when you add new themes.
 
 4. Update the palette number (key name =PaletteX=) accordingly. For example, if you have 1
-   custom color palette already, it should be =Pallet2= for the new one:
+   custom color palette already, it should be =Pallete2= for the new one:
```
Comment (shiitake) [inline README.org]: This should be spelled **Palette** (one L, two T's)

**Review by shiitake — CHANGES_REQUESTED** [review]
Palette is not spelled correctly.

**Comment by dhruvdutt** [comment]
Updated.

**Review by shiitake — APPROVED** [review]

**Comment by joonro** [comment]
Thanks a lot! I changed the base branch into `develop` and merged the PR.

**Pull request merged**